features-sdk/step_definitions/security-steps.js

```diff
@@ -59,7 +59,7 @@ Then('I am able to find {int} roles by searching controller:', async function (c
 Then('I am able to mGet roles and get {int} roles with the following ids:', async function (count, dataTable) {
   const data = this.parseObject(dataTable);
   const roleIds = [];
-  for (const role of Object.entries(data.ids)) {
+  for (const role of Object.values(data.ids)) {
```

Comment:
The `ids` arguments passed to `security:mGetRoles` was:
```js
{
  ids: [
    ['0', 'test-role'],
    ['1', 'test-role2'],
    ['2', 'test-role3']
  ]
}
```

Now that `security:mGetRoles` is stricter, this test fails with a `security.role.not_found` exception, because there is no role with the id `['0', 'test-role']`.